pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/v3/AdminApiTransactionMultiBrokerTest.java

```diff
@@ -81,4 +90,36 @@ public void testRedirectOfGetCoordinatorInternalStats() throws Exception {
             localAdmin.transactions().getCoordinatorInternalStats(i, false);
         }
     }
+
+    @Test
+    public void testGetTransactionBufferInternalStatsInMultiBroker() throws Exception {
+        for (int i = 0; i < super.getBrokerCount(); i++) {
+            getPulsarServiceList().get(i).getConfig().setTransactionBufferSegmentedSnapshotEnabled(true);
+        }
+        final String topic1 = NAMESPACE1 +  "/testGetTransactionBufferInternalStatsInMultiBroker";
+        while (true) {
+            admin.topics().createNonPartitionedTopic(topic1);
+            String segmentTopicBroker = admin.lookups()
+                    .lookupTopic(NAMESPACE1 + "/" + SystemTopicNames.TRANSACTION_BUFFER_SNAPSHOT_SEGMENTS);
+            String indexTopicBroker = admin.lookups()
+                    .lookupTopic(NAMESPACE1 + "/" + SystemTopicNames.TRANSACTION_BUFFER_SNAPSHOT_INDEXES);
+            if (segmentTopicBroker == indexTopicBroker) {
+                admin.topics().delete(topic1);
```

Comment:
Why deleting “topic1” affect the allocation of `TRANSACTION_BUFFER_SNAPSHOT_SEGMENTS ` and `TRANSACTION_BUFFER_SNAPSHOT_INDEXES`?

Reply:
Sorry, I make a mistake.